rest/taskana-rest-spring/src/test/java/pro/taskana/classification/rest/ClassificationDefinitionControllerIntTest.java

```diff
@@ -129,48 +127,48 @@ void testImportFilledClassification() throws Exception {
     classification.setCustom7("custom");
     classification.setCustom8("custom");
 
-    TaskanaPagedModel<ClassificationRepresentationModel> clList =
-        new TaskanaPagedModel<>(TaskanaPagedModelKeys.CLASSIFICATIONS, List.of(classification));
+    ClassificationCollectionRepresentationModel clList =
+        new ClassificationCollectionRepresentationModel(List.of(classification));
 
     ResponseEntity<Void> response = importRequest(clList);
     assertThat(response.getStatusCode()).isEqualTo(HttpStatus.NO_CONTENT);
   }
 
   @Test
-  void testFailureWhenKeyIsMissing() throws Exception {
+  void testFailureWhenKeyIsMissing() {
```

Comment:
doesn't fit our test naming schema...I know that this is not a new test, so feel free to ignore this comment ;)